src/components/views/DownloadModModal.vue

```diff
@@ -72,37 +70,13 @@ import ThunderstoreDownloaderProvider from '../../providers/ror2/downloading/Thu
         async downloadHandler(tsMod: ThunderstoreMod, tsVersion: ThunderstoreVersion) {
```

Comment:
Is there any reason for this to be defined in `DownloadModModal` anymore, instead of placing it directly in `DownloadModVersionSelectModal`. Is there any reason for `DownloadModVersionSelectModal` and `UpdateAllInstalledModsModal` to be subcomponents of this component, for that matter?

Just pointing this out, it can and probably should be handled in a separate PR.